apps/src/sites/studio/pages/layouts/_initial_section_creation_interstitial.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import InitialSectionCreationInterstitial from '@cdo/apps/templates/sectionSetup/InitialSectionCreationInterstitial';
+import {getStore} from '@cdo/apps/redux';
+import {Provider} from 'react-redux';
+import {trySetLocalStorage, tryGetLocalStorage} from '@cdo/apps/utils';
+
+const INITIAL_DIALOG_DISMISSED = 'initial_section_creation_dialog_dismissed';
+
+document.addEventListener('DOMContentLoaded', () => {
+  // Component should only be displayed on first page load of first log-in
+  // Set local storage and check whether component has been seen before.
+  if (tryGetLocalStorage(INITIAL_DIALOG_DISMISSED, 'false') === 'true') {
```

Comment:
This is a little confusing to read- do we need the "=== 'true'" for this to work?

Reply:
I believe we do - since tryGetLocalStorage(INITIAL_DIALOG_DISMISSED, 'false') returns the string 'true', we are comparing these as strings.

Reply:
gotcha- works for me then!